src/elements.rs

```diff
@@ -337,7 +370,7 @@ impl Element {
     #[cfg_attr(docsrs, doc(alias = "outerHTML"))]
     pub async fn html(&self, inner: bool) -> Result<String, error::CmdError> {
         let prop = if inner { "innerHTML" } else { "outerHTML" };
-        Ok(self.prop(prop).await?.unwrap())
+        Ok(self.prop(prop).await?.unwrap().to_string())
```

Comment:
I wonder if we should be a bit more principled here and instead of `.unwrap().to_string()` specifically match for `Some(Json::String` and error anything else?